docs/src/pages/ui/primitives/passwordfield/react.mdx

```diff
@@ -25,13 +25,15 @@ The PasswordField primitive allows users to input passwords. It also features fu
 
 ## Usage
 
+<Fragment>{() => import('./../shared/requiredAmplifyProvider.mdx')}</Fragment>
+
```

Comment:
What were your thoughts on my recommendation to document this once as part of the installation for React vs. on every component?
> https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/pages/ui/getting-started/installation/dependencies.react.mdx

Radix does the same – exclude the provider from primitive pages:
- https://www.radix-ui.com/docs/primitives/components/tabs
- https://www.radix-ui.com/docs/primitives/overview/server-side-rendering

Reply:
Yeah, after I pushed this I was thinking about that potentially being a better option. Will revisit this later today.

Reply:
@ericclemmons Updated.